src/main/java/com/mraof/minestuck/skaianet/InfoTracker.java

```diff
@@ -96,92 +99,100 @@ void requestInfo(ServerPlayer player, PlayerIdentifier p1)
 		
 		if(cannotAccess(player, p1))
 		{
-			player.sendSystemMessage(Component.literal("[Minestuck] ").withStyle(ChatFormatting.RED).append(Component.translatable(SkaianetHandler.PRIVATE_COMPUTER)));
+			player.sendSystemMessage(Component.literal("[Minestuck] ").withStyle(ChatFormatting.RED).append(Component.translatable(PRIVATE_COMPUTER)));
 			return;
 		}
 		if(!getSet(p1).add(p0))
 		{
 			LOGGER.warn("[Skaianet] Player {} already got the requested data.", player.getName());
 		}
 		
-		SkaianetInfoPacket packet = generateClientInfoPacket(p1);
-		MSPacketHandler.sendToPlayer(packet, player);
+		MSPacketHandler.sendToPlayer(generateClientInfoPacket(p1), player);
 	}
 	
 	
-	private SkaianetInfoPacket createLandChainPacket()
+	private SkaianetInfoPacket.LandChains createLandChainPacket()
 	{
-		return SkaianetInfoPacket.landChains(landChains.get());
+		return new SkaianetInfoPacket.LandChains(createLandChains());
 	}
 	
 	private List<List<ResourceKey<Level>>> createLandChains()
 	{
 		List<List<ResourceKey<Level>>> landChains = new ArrayList<>();
 		
 		Set<ResourceKey<Level>> checked = new HashSet<>();
-		skaianet.sessionHandler.getConnectionStream().forEach(c -> populateLandChain(landChains, checked, c));
+		skaianet.players().forEach(player -> populateLandChain(landChains, checked, player));
```

Comment:
A server side crash can occur here when /debuglands is used, the details are in discord